requirements.txt

```diff
@@ -1 +1,6 @@
+PyPrint>=0.2.0
+setuptools>=19.2
```

Comment:
FYI use `~=` here.

Also, coala should be here if coala-bears is. And vice versa (because coala-bears already has coala as a requirement).

I dont see why this depends on coala-bears.